### ISSUE TITLE
Doesn't work in android device

### ISSUE THREAD

**Issue by ntrp** [mjwheatley/Ionic2-Meteor1.3-Starter#1]
The application doesn't work if deployed into an android device. To test run the following:

```
meteor add-platform android
meteor run android-device
```

the received error is:

```
I20160408-00:52:28.964(2)? 04-08 00:51:30.664  9166  9166 I chromium: [INFO:CONSOLE(72763)] "Uncaught TypeError: Cannot read property 'classList' of null", source: http://localhost:12432/packages/modules.js?hash=666e5ea07ac834108e8ba7e91f39cee82f30ac74 (72763)
I20160408-00:52:29.078(2)? 04-08 00:51:30.778  9166  9166 I chromium: [INFO:CONSOLE(1185)] "deviceready has not fired after 5 seconds.", source: http://localhost:12432/cordova.js (1185)
```

**Comment by ntrp** [comment]
Found the problem, the initialization of the Ionic providers starts before meteor is ready. You can find a solution here:

https://github.com/ntrp/meteor-ionic-angular2-starter/blob/master/client/lib/meteor-ionic-app.ts

**Comment by mjwheatley** [comment]
I cloned your repo, but I could not get it running either.  I've been trying to upgrade this project to keep up with the rapid releases for Meteor 1.3, but more and more error keep cropping up.  Let me know if you are having any luck.

**Comment by ntrp** [comment]
Now it should be ok, I have set the dependencies to fixed. I have tried with a fresh cloned repository and it works.

**Comment by mjwheatley** [comment]
Thanks @ntrp,

I was able to clone and run your project.  I fixed my project and was able to run both of them on Android.  I could not get your project to build in Xcode however.  Mine did build, but did not show the content of my home page.

I fixed the SASS build for Ionic2 as well.  Before it was compiling both platform stylesheets into a single bundle and was causing some conflicts.  I changed it to build platform specific bundles and added code in the app.ts to detect which stylesheet should be active.  You will probably want to do the same in your project.
